thewalrus/quantum/fock_tensors.py

```diff
@@ -600,3 +601,33 @@ def _prefactor(mu, cov, hbar=2):
     beta = complex_to_real_displacements(mu, hbar=hbar)
     Qinv = np.linalg.inv(Q)
     return np.exp(-0.5 * beta @ Qinv @ beta.conj()) / np.sqrt(np.linalg.det(Q))
+
+def tvd_cutoff_bound(mu, cov, cutoff, hbar=2, check_is_valid_cov=True, rtol=1e-05, atol=1e-08):
+    r""" Gives a bound of the total variation instance between the exact Gaussian Boson Sampling
+    distribution extending to infinity in Fock space and the one truncated up to a certain cutoff.
+
+    For the derivation see Appendix 2 of `'Exact simulation of Gaussian boson sampling in polynomial space and exponential time',
```

Comment:
```suggestion
    For the derivation see Appendix B of `'Exact simulation of Gaussian boson sampling in polynomial space and exponential time',
```
Just checked the paper and appendices are labelled by letters